web/src/compose_reply.js

```diff
@@ -120,12 +124,19 @@ export function respond_to_message(opts) {
 
 export function reply_with_mention(opts) {
     respond_to_message(opts);
+    assert(message_lists.current !== undefined);
```

Comment:
Might be better to have the `assert` as the first line in the function; that feels more readable.

Reply:
Done.